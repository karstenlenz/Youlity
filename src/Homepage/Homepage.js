import React from 'react'
import { Link } from 'react-router-dom'
import styled from 'styled-components/macro'
import BrandHeadline from '../common/BrandHeadline'
import BulletList from '../common/BulletList'
import Button from '../common/Button'
import Card from '../common/Card'
import Collapse from '../common/Collapse'
import FloatingButtonContainer from '../common/FloatingButtonContainer'
import HeadlineUnderline from '../common/HeadlineUnderline'

export default function () {
  return (
    <>
      <img alt="" src="/img/mirror.svg" />
      <Logo alt="Youlity" src="/img/logo.svg" />
      <HeadlineUnderline>
        <h1>Discover Your Personality!</h1>
      </HeadlineUnderline>
      <p>
        Youlity ist eine Plattform, die Ihnen hilft, Ihre eigene Persönlichkeit
        über sogenannte Persönlichkeitsstile besser zu verstehen.
      </p>
      <Collapse headline="Was sind Persönlichkeitsstile?">
        <p>
          Die Psychologie hat sich lange hauptsächlich mit
          Persönlichkeits-störungen beschäftigt. Doch auch gesunde Menschen
          haben ein Interesse daran, Ihre Persönlichkeit zu verstehen, und in
          bestimmten Situationen ihr Handeln zu verändern. Dazu dient unser
          Youlity-Persönlichkeitstest. Im Gegensatz zu Tests wie z.B. „16
          Personalities“, die jeden Menschen in starre Persönlichkeitstypen
          zwängen wollen, erkennen wir an, dass jeder Mensch individuell ist.
          Deshalb messen wir Übereinstimmungen zu einzelnen
          „Persönlichkeitsstilen“. So entsteht ein differenziertes Modell Ihrer
          individuellen Persönlichkeit.
        </p>
      </Collapse>
      <Card>
        <BrandHeadline firstWord="Your" secondWord="Personality" />
        <SmallImg src="/img/style_info.svg" alt="" />
        <h2>Der Youlity-Test</h2>
        <BulletList listType="bullet_circle">
          <li>Entdecken Sie Ihre individuellen Persönlichkeitsstile.</li>
          <li>
            Entwickeln Sie ein besseres Verständnis für sich selbst in
            Beziehungen.
          </li>
          <li>Ideale Gesprächsstütze für Therapie-Termine. </li>
        </BulletList>
        <Link to="/motives/intro">
          <Button>Test starten</Button>
        </Link>
      </Card>
      <Card>
        <BrandHeadline firstWord="Your" secondWord="Reality" />
        <SmallImg src="/img/journal.svg" alt="" />
        <h2>Das Youlity-Tagebuch</h2>
        <BulletList listType="bullet_circle">
          <li>
            Notieren Sie sich Situationen, in denen sich Ihre Persönlichkeit
            widerspiegelt.
          </li>
          <li>Selbstbeobachtung ermöglicht Veränderung.</li>
          <li>Ideale Gesprächsstütze für Therapie-Termine.</li>
        </BulletList>
        <Link to="/journal">
          <Button>Tagebuch öffnen</Button>
        </Link>
      </Card>
      <BrandHeadline firstWord="Your" secondWord="Quality" />
      <HeadlineUnderline>
        <h2>Unser Versprechen</h2>
      </HeadlineUnderline>
      <BulletList listType="bullet_checkmark">
        <li>
          <h3>Seriös</h3>
          <p>
            Unser Angebot basiert auf neuesten Forschungsergebnissen der
            Psychologie.
          </p>
        </li>
        <li>
          <h3>Schnell</h3>
          <p>Der Test dauert nur ca. 5 Minuten.</p>
        </li>
        <li>
          <h3>Sicher</h3>
          <p>
            Wir übermitteln keine Daten von Ihnen. Ihre Privatsphäre ist absolut
            sicher.
          </p>
        </li>
      </BulletList>
      <FinePrint>
        Hinweis: Unser Angebot ersetzt keine psychologische Beratung. Wenn Sie
        unter psychischen Problemen leiden, suchen Sie sich bitte professionelle
        Hilfe.
      </FinePrint>
      <FloatingButtonContainer to="/motives/intro">
        <Button btnType="primary">Test starten</Button>
      </FloatingButtonContainer>
    </>
  )
}

const Logo = styled.img`
  margin-top: 20px;
  width: 50vw;
`
const SmallImg = styled.img`
  width: 50%;
`

const FinePrint = styled.small`
  margin-top: 20px;
  display: block;
`
