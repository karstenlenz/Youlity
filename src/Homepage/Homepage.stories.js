import React from 'react'
import Homepage from './Homepage'

export default {
  title: 'Homepage',
  component: Homepage,
}

export const HomepageComponent = () => <Homepage />
