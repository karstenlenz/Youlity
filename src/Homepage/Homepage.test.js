import { render } from '@testing-library/react'
import React from 'react'
import Homepage from './Homepage'

describe('The Homepage component', () => {
  it('renders correctly', () => {
    const { container } = render(<Homepage />)
    expect(container).toMatchSnapshot()
  })
})
