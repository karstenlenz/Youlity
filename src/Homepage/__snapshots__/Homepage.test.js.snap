// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Homepage component renders correctly 1`] = `
<div>
  <svg
    class="Homepage__MirrorStyled-ye7llx-1 bxPHyM"
    title=""
  >
    mirror.svg
  </svg>
  <svg
    class="Homepage__LogoStyled-ye7llx-0 csaKgP"
    title="Youlity"
  >
    logo.svg
  </svg>
  <section
    class="HeadlineUnderline__HeadlineContainer-sc-1reeb6v-0 iPfevw"
  >
    <h1>
      Discover Your Personality!
    </h1>
    <svg
      class="HeadlineUnderline__UnderlineStyled-sc-1reeb6v-1 iXilNe"
    >
      underline.svg
    </svg>
  </section>
  <p>
    Youlity ist eine Plattform, die Ihnen hilft, Ihre eigene Persönlichkeit über sogenannte Persönlichkeitsstile besser zu verstehen.
  </p>
  <section
    class="Collapse__CollapseWrapper-sc-1tgbx39-0 imOYVX"
  >
    <div
      class="Collapse__CollapseHeader-sc-1tgbx39-1 hKtGZf"
    >
      Was sind Persönlichkeitsstile?
      <img
        class="Collapse__CollapseIcon-sc-1tgbx39-2 bFSuxH"
        src="/img/expand.svg"
      />
    </div>
    <div
      class="Collapse__CollapseContent-sc-1tgbx39-3 domdvA"
    >
      <p>
        Die Psychologie hat sich lange hauptsächlich mit Persönlichkeits-
        <em>
          störungen
        </em>
         beschäftigt. Doch auch gesunde Menschen haben ein Interesse daran, Ihre Persönlichkeit zu verstehen, und in bestimmten Situationen ihr Handeln zu verändern. Dazu dient unser Youlity-Persönlichkeitstest.
        <br />
         Im Gegensatz zu Tests, die mit starren Persönlichkeits-
        <em>
          Typen
        </em>
         arbeiten, wie z. B. „16 Personalities“, gehen wir davon aus, dass jeder Mensch individuell ist. Deshalb messen wir nur Übereinstimmungen zu einzelnen „Persönlichkeitsstilen“. Es gibt insgesamt 9 verschiedene Persönlichkeitsstile.
        <br />
        Persönlichkeitsstile sind Muster aus stabilen Merkmalen in Menschen, die sich aus der individuellen Lebensgeschichte ergeben. Youlity hilft Ihnen, ein differenziertes Modell Ihrer individuellen Persönlichkeit zu erstellen.
      </p>
    </div>
  </section>
  <div
    class="Card__CardBG-sc-1k8174e-0 bqDKbN"
  >
    <div
      class="BrandHeadline__Wrapper-sc-1gsl6zz-0 kOqNrf"
    >
      <h3
        class="BrandHeadline__Green-sc-1gsl6zz-1 ljEpsh"
      >
        Your
          
      </h3>
       
      <h3
        class="BrandHeadline__Purple-sc-1gsl6zz-2 Kbtt"
      >
        Personality
      </h3>
    </div>
    <svg
      class="Homepage__StyleInfoImgStyled-ye7llx-2 fxhNWA"
      title=""
    >
      style_info.svg
    </svg>
    <h2>
      Der Youlity-Test
    </h2>
    <ul
      class="BulletList__ListContainer-sc-1r3lk85-0 FqDGi"
    >
      <li>
        Entdecken Sie Ihre individuellen Persönlichkeitsstile.
      </li>
      <li>
        Entwickeln Sie ein besseres Verständnis für sich selbst in zwischenmenschlichen Beziehungen.
      </li>
      <li>
        Profitieren Sie von speziell auf Sie zugeschnittenen Tipps für Alltagssituationen.
      </li>
    </ul>
    <a
      href="/motives/intro"
    >
      <button
        class="Button__BtnStyled-sc-129yfz5-0 bJYIPU"
        width="100"
      >
        Test starten
      </button>
    </a>
  </div>
  <div
    class="Card__CardBG-sc-1k8174e-0 bqDKbN"
  >
    <div
      class="BrandHeadline__Wrapper-sc-1gsl6zz-0 kOqNrf"
    >
      <h3
        class="BrandHeadline__Green-sc-1gsl6zz-1 ljEpsh"
      >
        Your
          
      </h3>
       
      <h3
        class="BrandHeadline__Purple-sc-1gsl6zz-2 Kbtt"
      >
        Reality
      </h3>
    </div>
    <svg
      class="Homepage__JournalImgStyled-ye7llx-3 fBdOhN"
      title=""
    >
      journal.svg
    </svg>
    <h2>
      Das Youlity-Tagebuch
    </h2>
    <ul
      class="BulletList__ListContainer-sc-1r3lk85-0 FqDGi"
    >
      <li>
        Notieren Sie sich Situationen, in denen sich Ihre Persönlichkeit widerspiegelt.
      </li>
      <li>
        Übernehmen Sie die Kontrolle: Selbstbeobachtung ermöglicht Veränderung.
      </li>
      <li>
        Nutzen Sie das Tagebuch als Gesprächsstütze für Therapie-Termine.
      </li>
    </ul>
    <a
      href="/journal"
    >
      <button
        class="Button__BtnStyled-sc-129yfz5-0 bJYIPU"
        width="100"
      >
        Tagebuch öffnen
      </button>
    </a>
  </div>
  <div
    class="BrandHeadline__Wrapper-sc-1gsl6zz-0 kOqNrf"
  >
    <h3
      class="BrandHeadline__Green-sc-1gsl6zz-1 ljEpsh"
    >
      Your
        
    </h3>
     
    <h3
      class="BrandHeadline__Purple-sc-1gsl6zz-2 Kbtt"
    >
      Quality
    </h3>
  </div>
  <section
    class="HeadlineUnderline__HeadlineContainer-sc-1reeb6v-0 iPfevw"
  >
    <h2>
      Unser Versprechen
    </h2>
    <svg
      class="HeadlineUnderline__UnderlineStyled-sc-1reeb6v-1 iXilNe"
    >
      underline.svg
    </svg>
  </section>
  <ul
    class="BulletList__ListContainer-sc-1r3lk85-0 kxqqwb"
  >
    <li>
      <h3>
        Seriös
      </h3>
      <p>
        Unser Angebot basiert auf neuesten Forschungsergebnissen der Psychologie.
      </p>
    </li>
    <li>
      <h3>
        Schnell
      </h3>
      <p>
        Der Test dauert nur ca. 5 Minuten.
      </p>
    </li>
    <li>
      <h3>
        Sicher
      </h3>
      <p>
        Wir übermitteln keine Daten von Ihnen. Ihre Privatsphäre ist absolut sicher.
      </p>
    </li>
  </ul>
  <small
    class="FinePrint__SmallStyled-sc-1oy87bz-0 bIdxte"
  >
    Hinweis: Unser Angebot ersetzt keine psychologische Beratung. Wenn Sie unter psychischen Problemen leiden, suchen Sie sich bitte professionelle Hilfe.
  </small>
  <a
    class="FloatingButtonContainer__ButtonContainer-slkv4p-0 fIGCfd"
    href="/motives/intro"
  >
    <button
      class="Button__BtnStyled-sc-129yfz5-0 bJYIPU"
      width="100"
    >
      Test starten
    </button>
  </a>
  <div
    class="FloatingButtonContainer__BottomSpacer-slkv4p-1 hgDFeh"
  />
</div>
`;
